### ISSUE TITLE
kernel-5.10 + 20210223 + update = ChromeOS not found

### ISSUE THREAD

**Issue by lfom** [sebanc/brunch#917]
Hello!

ChromeOS does not load anymore, it stops before loading the splash screen. I have Brunch installed since some time ago, all updates worked fine so far. I was using kernel-5.10.

I had Brunch with ChromeOS v86 installed on an image in a partition of my internal storage
1. Downloaded and updated Brunch to latest release (20210223)
2. Installed and rebooted, partition was updated fine, no errors
3. I checked Brunch version and it did show 20210223
4. I went to Settings and updated ChromeOS to v88
5. Upon restart it showed an error saying something like "ChromeOS was not found in any device" and loaded a shell

I still can access and mount the image from Linux, the only thing weird I noticed is that it does not have a ROOT-B partition, but two ROOT-A and one ROOT-C. Actually I have noticed it some time ago, thought it was weird and that I would report it, but didn't do it in the end.

Intel Atom X5 convertible

_Update1_: I have edited GRUB and changed it to use the default kernel, and it did the Brunch update stuff and it booted fine again. I was able to confirm it is running ChromeOS v88 and kernel 5.4.88. I will try to go back to 5.10 now.
_Update2_: it worked, now booted to 5.10.6.

**Comment by Jerit3787** [comment]
Did it solve your problem?

**Comment by lfom** [comment]
@Jerit3787 Yeah, as I reported above: first reverted to default kernel (5.4) and it installed Brunch after the update. Then went back to kernel 5.10, it did his thing again, everything seemed fine.

**Comment by Jerit3787** [comment]
alright so please consider closing this thread. thank you
